app/database_etl/etl_main.py

```diff
@@ -34,6 +34,8 @@ def read_from_json(path_to_json):
 dirname = os.path.dirname(__file__)
 filename = os.path.join(dirname, 'country_iso3.json')
 ISO3_CODES = read_from_json(filename)
+filename = os.path.join(dirname, 'country_iso2.json')
```

Comment:
can we refactor these lines? make them a function where u pass in the file name or smth so it's more organized?

Reply:
sure